docs/RELEASE-NOTES.rst

```diff
@@ -1,38 +1,55 @@
 Release Notes for Container Ingress Services for Kubernetes & OpenShift
 =======================================================================
 
-Next Release
-------------
+2.5.0
+-------------
 
 Added Functionality
 ```````````````````
-* Added support for networking.k8s.io/v1 ingress and ingressClass.
-* Added support for "virtual-server.f5.com/clientssl" annotation in ingress resource.
-* Service Type LB Enhancements:
-   - Added support for Multiport service.
-   - Added support for HealthMonitor
-* Added support to send telemetry data to F5 TEEM Server 
-   - Disable this feature optionally using configuration parameter `disable-teems:"true"`
-* Added new deployment parameter `periodic-sync-interval` for configuring the periodic sync of resources
-* Added informers for kubernetes secrets to monitor the changes in secrets
-* Added support for configmaps to monitor services in same and different namespaces.
-    - Enable this support using deployment parameter `hubmode:true`
-* Helm Chart Enhancements:
-   - Added ingressClass resource installation with charts
-* Added support for AS3 3.28
+* CIS now compatible with:
+    - Kubernetes 1.21
+    - OpenShift 4.7.13 with OpenShift SDN
+    - AS3 3.28
 
+* Added support for:
+   - :issues: `1824` Ingress and IngressClass objects in networking.k8s.io/v1. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/ingress/networkingV1>`_.
```

Comment:
We can add :

With networking.k8s.io/v1 ingress, Add multiple BIGIP SSL client profiles with annotation "virtual-server.f5.com/clientssl". Refer for Examples <>

Reply:
Done